docs/README.md

```diff
@@ -183,19 +183,93 @@ end
 You can also override an entire controller by simply placing a file with the controller's name in the controllers directory.  For example, `app/controllers/general_controller.rb` would take the place of the Orchid version of this file.  *This approach is not recommended.*
 
 ### Facets
-You may also want to peruse the `app/models/facets.rb` file and alter it for specific fields which you would prefer.  If your app supports multiple languages, you will need to make sure that the facets are organized by language, for example:
+Facets are API fields used to index items in the "Browse" pages. For each
```

Comment:
I find this phrasing a little confusing: `used to index items`, because that makes it sound like they are somehow populating the API to me.  What about something like `Facets are API fields that aggregate results by shared metadata, for example, a "category" listing on the "Browse" page.`  Okay, I don't like that, either....um, any other ideas?

Reply:
Will push a change after discussing how to avoid confusing language `used to index items` :+1: